KP5-Topic2/KP4-Topic2/target/KP4-Topic2/index.jsp

```diff
@@ -0,0 +1,31 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"

```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files